Dockerfile

```diff
@@ -85,8 +89,9 @@ RUN cd ${TEMP} \
 && rm -rf ${TEMP} \
 \
 # Add group and user
-&& groupadd -g ${GID} nexus \
-&& adduser -u ${UID} -d ${IQ_HOME} -c "Nexus IQ user" -g nexus -s /bin/false nexus \
+&& addgroup -g ${GID} nexus \
```

Comment:
addgroup and adduser commands are used in Alpine instead of groupadd and useradd. Also, the -D option is used with adduser instead of -d, and /sbin/nologin is used instead of /bin/false as the login shell. Finally, the chown command is used to set ownership of the IQ home directory to the nexus user and group